src/ownership-and-deconstruction.rst

```diff
@@ -20,7 +20,7 @@ Ownership
 management model of Rust.
 
 :dp:`fls_ckcnkbb6y3cq`
-An :t:`owner` is an :t:`object` that holds a :t:`value`.
+An :t:`owner` is a :t:`variable` that holds a :t:`value`.
```

Comment:
This should state `value holder` instead of variable actually I think. A `static` can very much own a value (it's just never dropped), a const technically as well (though there given its usages is basically placed at the use site it doesn't really use ownership actively).

We might wanna split that off into an issue though, not sure how the chapter talks about borrowing in regards to the term `owner` (since statics are a bit different in regards to borrowing, mutable borrows needing unsafe etc)

Reply:
Will use `variable holder` and spawn off a separate issue to revisit `variable holder` and `owner`.